Library/Homebrew/cmd/update-report.rb

```diff
@@ -150,6 +151,15 @@ def update_report
       puts "Already up-to-date." unless args.quiet?
     end
 
+    if Completions.read_completions_option.empty?
```

Comment:
Will this show once or every time? Should this be limited to only if there are available completions to be linked?

Reply:
Yes, this will only show one time. It will show when the `homebrew.linkcompletions` git config value is empty. Then it will write `no` so the message won't show in the future.